deliverables/README.md

```diff
@@ -1,68 +1,140 @@
-# Week 1 Deliverable  
+# Week 1 Deliverable
 
-### Instructions  
+### Instructions
 
-In order to deliver homework consisting of questions and answers, just edit this file and commit the changes.  Once your answers are on Github, specifically in this file, you have delivered your homework.  
-  
-To answer questions, please just put the answer on the next line after the question.  Put a blank line after the answer to separate it visually from the next question.  
+In order to deliver homework consisting of questions and answers, just edit this file and commit the changes.  Once your answers are on Github, specifically in this file, you have delivered your homework.
 
-Example:  
+To answer questions, please just put the answer on the next line after the question.  Put a blank line after the answer to separate it visually from the next question.
+
+Example:
 
 **What is your name?**
-Sir Student of Ruby
+Christian Dean
 
 **What is your quest?**
-I seek the Rails  
+I want to learn more about code and how to utilize tools to make my business succeed.
 
 **What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+Green is my favorite color!
 
-For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
+For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.
 
 ### Section 1: The Front End
-#### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+#### Required - Answer these questions in the deliverables file, and complete the following.
+
+What is the role of HTML in a web page?
+HTML is the language, or code, used to edit and position the text, images, frames and other web page elements. Its purely for content.
+What is the role of CSS in a web page?
+CSS is an enhanced form of HTML, and makes it easier to edit the design of a web page.
```

Comment:
It's not an enhanced form of HTML; it's another component that works with, but is separate from, HTML